static-webp/src/main/jni/third-party/libjpeg-turbo-1.5.3/config.h

```diff
@@ -110,18 +110,19 @@
 #define PACKAGE_URL ""
 
 /* Define to the version of this package. */
-#define PACKAGE_VERSION "1.5.0"
+#define PACKAGE_VERSION "1.5.3"
 
 /* Define if your (broken) compiler shifts signed values as if they were
    unsigned. */
 /* #undef RIGHT_SHIFT_IS_UNSIGNED */
-
+// TODO: double check if it was done on purpose. jconfig.h has different defines for 64 and 32 bits
```

Comment:
I wonder if it is intentionally was changed to 1.

The same thing is defined as
```
#ifdef __LP64__
#  define SIZEOF_SIZE_T 8
#else
#  define SIZEOF_SIZE_T 4
#endif
```